pkgs/build-support/writers/test.nix

```diff
@@ -61,17 +60,6 @@ let
       print "success\n" if true;
     '';
 
-    pypy2 = writePyPy2Bin "test-writers-pypy2-bin" { libraries = [ pypy2Packages.enum ]; } ''
```

Comment:
If we are going to remove the `pypy2` tests we should also remove `pypy3`. `pypy3` also depends on `python2` (or in `pypy2` after #209100 merge).

It is not like `pypy2` is being deprecated though, and with #209100 those tests should pass without issue. So I would drop this commit.

Reply:
Dropped that commit, misunderstood the python2/pypy2 relationship.